app/src/main/java/com/kickstarter/libs/ActivityViewModel.java

```diff
@@ -23,8 +23,9 @@
   private final CompositeSubscription subscriptions = new CompositeSubscription();
 
   private final PublishSubject<ActivityResult> activityResult = PublishSubject.create();
-
   private final PublishSubject<Intent> intent = PublishSubject.create();
+  protected final PublishSubject<ActivityEvent> onResume = PublishSubject.create();
```

Comment:
sorry, this was my suggestion, but after seeing it IRL now I think we should revert and go back to the explicit `resume` input you had before. I really don't wanna push more functional into this VM base class. there's a lot of benefit to making our VM's very explicit in what they require, so it's nice to see this VM say it has functionality built on top of the activity resuming. 

do you mind reverting this commit? you can just do a `git revert` to do it cleanly. 

also you can make the resume observable just a simple `<Void>` observable since we no longer need the activity lifecycle event.